src/features/game/expansion/lib/boosts.test.ts

```diff
@@ -1,34 +0,0 @@
-import Decimal from "decimal.js-light";
```

Comment:
Redundant tests, that logic is covered in src\features\game\events\landExpansion\fulfillGrubOrder.test.ts